cfn-resources/cloud-backup-snapshot-export-job/test/key-policy-template.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
Do we really need these many permissions for this policy?

Reply:
removed this file